### ISSUE TITLE
Будет ли обновления? 

### ISSUE THREAD

**Issue by dcbeelinekz** [kraiz/eiskaltdcpp-daemon-openwrt#2]

**Comment by kraiz** [comment]
Hey, I'm currently not using it anymore but could maybe help if you try to update.

**Comment by dcbeelinekz** [comment]
нет. спасибо дорогой друг, я в этом не силен, дуал может ты сделал что то новее . Спасибо

**Comment by kraiz** [comment]
Maybe not published to the public package list. Have a look at the release page: https://github.com/makefu/eiskaltdcpp-daemon-openwrt/releases

My current setup is docker-based: https://github.com/kraiz/icecold-peered

**Comment by dcbeelinekz** [comment]
Дорогой друг, kraiz , а можешь проверить на LEDE сам, если у тебя есть время на это. спасибо
